client/jetpack-connect/style.scss

```diff
@@ -882,6 +882,57 @@ $colophon-height: 50px; // wpcomColophon element at the bottom
 	/** WooCommerce Onboarding Styles **/
 	&.is-jetpack-woocommerce-flow {
 		background-color: var( --color-woocommerce-onboarding-background );
+
+		.jetpack-connect__main-logo {
+			text-align: center;
+			border-bottom: 1px solid #1f112e;
```

Comment:
Hardcoded hex, see https://github.com/Automattic/wp-calypso/pull/33073/files#r284687907

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/33073/commits/ec4b2b0da4a0c1137a0b4760fb87999cff3137aa.